about.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles/style.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+    <title>About</title>
+</head>
+<body>
+    <section class="about" id="about">
+        <div class="mainImg">
+            <nav class="navbar fixed-top topNav">
+                <nav class="heading">
+                  <img class="heading-elements" src="images/facebook.jpg" width="24" height="24" alt="facebook">
+                  <img class="heading-elements" src="images/twitter.png" width="24" height="24" alt="twitter">
+                  <span class="text-white fw-bold heading-elements">English</span>
+                  <span class="text-white fw-bold heading-elements">My Page</span>
+                  <span class="text-white fw-bold heading-elements">Logout</span>
```

Comment:
### Your Project
![missing-navlinks (copy)](https://user-images.githubusercontent.com/93039359/181393899-abefa6a6-95f0-49aa-bf1c-7548aac4cd38.png)

### Design
![design-nav-about](https://user-images.githubusercontent.com/93039359/181393913-fc4f60dd-c86c-431b-8082-07b7196eaf8a.png)

- [ ] In your about page your navigation links are not visible please add these links as shown in the Design Template :blush: